CPFTreasury/build.gradle

```diff
@@ -0,0 +1,53 @@
+version = '0.9.1'
+
+dependencies {
+    compileOnly 'foundation.icon:javaee-api:0.9.1'
+    implementation 'com.github.sink772:javaee-tokens:0.6.1'
+    implementation 'com.github.sink772:minimal-json:0.9.6'
+
+    testImplementation 'foundation.icon:javaee-unittest:0.9.2'
+    implementation 'org.mockito:mockito-core:4.3.1'
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.2'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.2'
+    testImplementation('org.mockito:mockito-inline:4.3.1')
+    implementation 'foundation.icon:javaee-scorex:0.5.2'
```

Comment:
Put implementation libraries first and then test implementation libraries and then inttest libraries. Are all these libraries necessary for main contract??